README.md

```diff
@@ -16,21 +19,84 @@ The Custom Metadata loader tool now supports updates of existing custom metadata
+   These users also need the 'Customize Application' to create Custom Metadata records. Admin should have this permission by default.
+2. To apply the permission set - CustomMetadataLoader to the user who is using the tool. Go to Administer->Manage Users ->Permission Sets. Click on Custom Metadata Loader.
 3  You will be taken to Permission Set page - Click on Manage Assignments. Then click Add Assignments. Choose the user/users. Then click Assign. Then Done. Now the perm set should be successfully assigned.
-4. Create a CSV file with a header that contains the field API names, including the org namespace. Either Label or Developer Name is required. A sample csv for CountryMapping__mdt is in the same folder as this README file. 
-5. Next you are ready to use the tool - Select Custom Metadata Loader from the app menu in your org, then go to the Custom Metadata Loader tab.The app will prompt you to create a remote site setting if it is missing. 
+4. Create a CSV file with a header that contains the field API names, including the org namespace. Either Label or Developer Name is required. A sample csv for CountryMapping__mdt is in the same folder as this README file.
+5. Next you are ready to use the tool - Select Custom Metadata Loader from the app menu in your org, then go to the Custom Metadata Loader tab.The app will prompt you to create a remote site setting if it is missing.
 6. Select the CSV file and the corresponding custom metadata type.
 7. Click 'Create/Update custom metadata' to bulk load the records from the CSV file into your org.
+
+# How to use custom metadata migrator
+
+Use one of the below option to migrate Custom Settings or Custom Objects to Custom Metadata Types. Go to the 'Custom Metadata Migrator' tab
+
+### Option 1: Migrate Custom Settings/Custom Objects to new Custom Metadata Type
+
+Input the following:
+
+	--Api name of Custom Setting or Custom Object (e.g. VAT_Settings_CS__c)
+	--Api name of Custom Metadata Types (e.g. VAT_Settings__mdt)
+
+Click on 'Migrate'
+
+### Option 2: Migrate Custom Settings/Custom Objects to existing Custom Metadata Type
+
+Input the following:
+
+	--Api name of Custom Setting (e.g. VAT_Settings_CS__c)
+	--Select the name of existing Custom Metadata Types
+
+Click on 'Migrate'
+
+### Option 3: Migrate Custom Settings/Custom Objects to existing Custom Metadata Type (using simple mapping)
+
+Input the following:
+
+	--Api name of Custom Setting.fieldName (e.g. VAT_Settings_CS__c.Active__c)
+	--Api name of Custom Metadata Types.fieldName (e.g. VAT_Settings__mdt.Active__c)
+
+Click on 'Migrate'
+
+### Option 4: Migrate Custom Settings/Custom Objects to existing Custom Metadata Type (using custom mapping)
+
+Input the following:
+
+	--Api Name of Custom Setting (e.g. VAT_Settings_CS__c)
+	--Api Name of Custom Metadata Types (e.g. VAT_Settings__mdt)
+	--Json Mapping (Sample below)
```

Comment:
Can you please clarify what the keys and values are? Just to say that the key should be the Custom Setting/Object field name and that the value is the CMT field name.